test/testdata/knownExtensions.ts

```diff
@@ -36,6 +37,10 @@ const knownExtensions = [
     ...VUEJS_DEVTOOLS,
     description: 'Vue.js devtools',
   },
+  {
+    ...VUEJS3_DEVTOOLS,
+    description: 'Vue.js devtools',
```

Comment:
```suggestion
    description: 'Vue.js 3 devtools',
```

Reply:
It's invalidate the test file, given description should be same the the extension description

https://github.com/MarshallOfSound/electron-devtools-installer/blob/19fd4dde8a8e3eb8a50ba4ca242bf3b52e878436/test/install_spec.ts#L72